venues/NeurIPS.cc/2021/Conference/python/build-conflicts.py

```diff
@@ -0,0 +1,227 @@
+
+    for submission in submissions:
+        author_ids=submission.details['original']['content']['authorids']
+
+        author_domains = set()
+        author_emails = set()
+        author_relations = set()
+
+        for author in author_ids:
+            author_info = get_profile_info(author)
+            author_domains.update(author_info['domains'])
+            author_emails.update(author_info['emails'])
+            author_relations.update(author_info['relations'])
+
+        for user in match_group.members:
+            user_info = get_profile_info(user)
+            conflicts = set()
+            conflicts.update(author_domains.intersection(user_info['domains']))
+            conflicts.update(author_relations.intersection(user_info['emails']))
+            conflicts.update(author_emails.intersection(user_info['relations']))
+            conflicts.update(author_emails.intersection(user_info['emails']))
+
+            if conflicts:
+                edges.append(openreview.Edge(
+                    invitation=invitation.id,
+                    head=submission.id,
+                    tail=user_info['id'],
+                    weight=-1,
+                    label='Conflict',
+                    readers=_get_edge_readers(tail=user_info['id']),
+                    writers=[conference.id],
+                    signatures=[conference.id]
+                    ))
+
+    openreview.tools.post_bulk_edges(client, edges=edges)
+
+    # Perform sanity check
+    edges_posted = client.get_edges_count(invitation=invitation.id)
+    if edges_posted < len(edges):
+        raise openreview.OpenReviewException('Failed during bulk post of Conflict edges! Scores found: {0}, Edges posted: {1}'.format(len(edges), edges_posted))
+    
+    return edges
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--baseurl', help="base url")
+    parser.add_argument('--username')
+    parser.add_argument('--password')
+
+    args = parser.parse_args()
+
+    client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+    conference=openreview.helpers.get_conference(client, '5-TLJ7mpH8g') ## neurips2021 dev site id
+
+    submissions = list(openreview.tools.iterget_notes(conference.client,
+                    invitation=conference.get_blind_submission_id(), details='original'))
+
+    sac_group=client.get_group(conference.get_senior_area_chairs_id())
+    ac_group=client.get_group(conference.get_area_chairs_id())
+    rev_group=client.get_group(conference.get_reviewers_id())
+
+    for match_group in [sac_group, ac_group, rev_group]:
+        is_area_chair = conference.get_area_chairs_id() == match_group.id
+        is_senior_area_chair = conference.get_senior_area_chairs_id() == match_group.id
+        should_read_by_area_chair = conference.get_reviewers_id() == match_group.id
+        post_edges=build_conflicts(match_group)
```

Comment:
it seems the submissions parameter is missing here?

Reply:
fixed